config/secrets.yml

```diff
@@ -17,6 +17,15 @@
 
 # Environmental secrets are only available for that specific environment.
 
+shared:
+  host: <%= ENV['GO_FOX_HOST'] %>
```

Comment:
Why

Reply:
Thanks @spencerldixon I put this in because it was present on a different project which I was learning from but I can remove it if we don't need it. Actually I noticed now that Future of Conservation does not have this either.